ACM_General/home/templates/home/base.html

```diff
@@ -1,128 +1,127 @@
-                      <img src="{% static "home/img/core/home-icon.png" %}"/>
-                      <h3>HOME</h3>
-                    </a>
-                    <a href="{% url "home:sponsors" %}" class="sponsors-nav nav-button">
-                      <img src="{% static "home/img/core/sponsors-icon.png"%}"/>
-                      <h3>SPONSORS</h3>
-                    </a>
-                    <a href="{% url "home:calendar" %}" class="calendar-nav nav-button">
-                      <img src="{% static "home/img/core/calendar-icon.png"%}"/>
-                      <h3>CALENDAR</h3>
-                    </a>
-                    <a href="{% url "home:media" %}" class="media-nav nav-button">
-                        <img src="{% static "home/img/core/media-icon.png"%}"/>
-                        <h3>MEDIA</h3>
-                    </a>
-                    <a href="{% url "home:officers" %}" class="officers-nav nav-button">
-                        <img src="{% static "home/img/core/officers-icon.png" %}"/>
-                        <h3>OFFICERS</h3>
-                    </a>
-                    <a href="{% url "thirdparty_auth:login" "oauth2" "google" %}" class="login-nav nav-button">
-                      <img src="{% static "home/img/core/login-icon.png"%}"/>
-                      <h3>LOGIN</h3>
-                    </a>
+  <header>
+    <div class="constraint-wrapper">
+      <div class="header-acm-seal-wrapper">
+        <div class="header-acm-seal">
+          <img src="{% static "/home/img/core/acm.png" %}">
+          <h3>Association for<br>Computing Machinery</h3>
+        </div>
+      </div>
 
-                </nav>
-            </div>
-        </header>
-{% endblock %}
+      <div class="header-welcome-wrapper">
+        <h1 class="header-welcome">We are Missouri S&T - ACM</h1>
+      </div>
+    </div>
+  </header>{% endblock %}
+{% block body_navigation %}
+  <div class="top-bar"></div> <!-- mobile menu bar -->
+  <div id="menu-wrapper"><!-- mobile menu -->
+    <input type="checkbox" id="menu" name="menu" class="menu-checkbox">
+    <div class="menu">
+      <label class="menu-toggle" for="menu"><span>Toggle</span></label>
+      <ul>
+        <li>
+          <a href="{% url "home:index" %}">Home</a>
+        </li>
+        <li>
+          <a href="{% url "home:sponsors" %}">Sponsors</a>
+        </li>
+        <li>
+          <a href="{% url "home:calendar" %}">Calendar</a>
+        </li>
+        <li>
+          <a href="{% url "home:media" %}">Media</a>
+        </li>
+        <li>
+          <a href="{% url "home:officers" %}">Officers</a>
+        </li>
+        <li>
+          <a
+            href="{% url "thirdparty_auth:login" "oauth2" "google" %}">Login</a>
+        </li>
+      </ul>
+    </div>
+  </div><!-- mobile menu -->
 
+  <div class="pseudo-nav"></div>
+  <nav><!-- desktop nav -->
+    <ul class="nav-link-wrapper">
+      <li class="nav-home nav-button"><a href="{% url "home:index" %}"
```

Comment:
Any way to format these as multiple lines? I don't think it looks good stylistically